integration-tests/kafka-avro/src/test/java/io/quarkus/it/kafka/KafkaAndSchemaRegistryTestResource.java

```diff
@@ -29,7 +29,7 @@ public static String getApicurioSchemaRegistryUrl() {
     @Override
     public Map<String, String> start() {
         kafka.start();
-        registry = new GenericContainer<>("apicurio/apicurio-registry-mem:1.2.2.Final")
+        registry = new GenericContainer<>("apicurio/apicurio-registry-mem:2.1.3.Final")
```

Comment:
Not here, this module intentionally tests with Apicurio Registry 1.x.

Instead, the `kafka-avro-apicurio2` test suite module needs a change in `pom.xml`.

Reply:
@carlesarnal can you handle Ladislav comment?